MoltenVK/MoltenVK/GPUObjects/MVKVulkanAPIObject.h

```diff
@@ -103,7 +103,7 @@ class MVKVulkanAPIObject : public MVKConfigurableObject {
 protected:
 	virtual void propagateDebugName() = 0;
 
-	std::atomic<uint32_t> _refCount;
+	std::atomic<uint64_t> _refCount;
```

Comment:
Is this change a requirement for the performance boost? Does the use of 64 bits improve access performance?

This change bumps the size of the base `MVKVulkanAPIObject` class up by 8 bytes (and leaves a 4-byte gap), which affects a lot of other classes and their instances. 

Not a huge deal, since these are API classes, so there will not be enormous quantities, and a 4% performance boost is definitely more important...but nevertheless, we've just finished a round of memory optimization on the class hierarchy, and I'd only like to regress on that if necessary.

Reply:
I tried testing this, and found no difference with 32-bit vs 64-bit.